global.json

```
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.100",
+    "version": "7.0.*",
```

This is going to fail with CodeQL, as it again doesn't yet fully support dotnet 7. It is better to add explicit version now and fall back to wildcards later when they add it.
```suggestion
    "version": "7.0.100",
```